### PR TITLE
Feature listado de mis alertas 161118271

### PR DESCRIPTION
**Pull request by FedePe**
Create pr corresponding to the history "list my alerts".
In this story, the stories created by the user are listed.

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -48,6 +48,9 @@ Home
                   <li className="drop-down">
                     <Link to="/profile" className="nav-link">My Profile</Link>
                   </li>
+                  <li className="drop-down">
+                    <Link to="/authored-alerts" className="nav-link">Authored Alerts</Link>
```
[the remark this answers is not in the document]
Comment (FedePe) [reply src/components/Header.jsx]: Done!

File: src/components/AuthoredAlert.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+import '../css/AuthoredAlert.css';
+import React from 'react';
+
+export default function Alert(props) {
```
[the remark this answers is not in the document]
Comment (FedePe) [reply src/components/AuthoredAlert.jsx]: You are right!

File: src/components/ViewAuthoredAlerts.jsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        </div>
+      );
+    }
+    return (
+      <div className="view-authored-alerts">
+        <div className="container-fluid">
+          <div className="row">
+            <div className="col-md-6 offset-md-3">
+              <div className="content-alerts list-group authored-alert">
+                {alerts.map(alert => (
```
[the remark this answers is not in the document]
Comment (FedePe) [reply src/components/ViewAuthoredAlerts.jsx]: You are right!

File: src/components/wyehome/MyAlert.jsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+import '../../assets/css/MyAlert.css';
+import React from 'react';
+
+export default function MyAlert(props) {
```
Comment (rocespedes) [inline src/components/wyehome/MyAlert.jsx]: i think the 'M' should be in lower case. It's only a comment.

**Pull request merged**